.github/workflows/build.yml

```diff
@@ -29,6 +29,8 @@ jobs:
           - boardsource3x4
           - corne_left
           - corne_right
+          - fifi_left
+          - fifi_right
```

Comment:
Please don't edit this file for now. We're working on removing this file in favor of a dynamic build script that doesn't require explicit builds.